nerfstudio/pipelines/base_pipeline.py

```diff
@@ -370,7 +370,7 @@ def load_pipeline(self, loaded_state: Dict[str, Any], step: int) -> None:
             loaded_state: pre-trained model state dict
             step: training step of the loaded checkpoint
         """
-        state = {key.replace("module.", ""): value for key, value in loaded_state.items()}
+        state = {key[len("module."):]: value for key, value in loaded_state.items()}
```

Comment:
I don't think this will work. You can have keys without the "module." prefix, right?

Reply:
you're right, just pushed a commit to fix this